### PR TITLE
lesson_01: lesson 01 exercises

### PR DESCRIPTION
**Pull request by sendeand**
6 exercises of Snakify/Lesson 1
9 exercises of Snakify/Lesson 2

### CODE REVIEW

**Comment by sendeand** [comment]
@kuszcjan Please check may changes for lesson_01

File: students/sendecki_andrzej/lesson_01_basics/clock_face-1.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+print("Enter the number of minutes")
+m = int(input())
+# it's a 12*60th (720) part of 360 degree
+angle_m = (m/720)*360
+
+print("Enter the number of seconds")
+s = int(input())
+# it's a 12*60*60th (43200) part of 360 degree
+angle_s = (s/43200)*360
```
Comment (kuszcjan) [inline students/sendecki_andrzej/lesson_01_basics/clock_face-1.py]: Good, there is a comment which describes a magic number.
The pythonic way to do such things is to write those as:
`angle_s = (s/12*60*60)*360`

File: students/sendecki_andrzej/lesson_01_basics/first_digit_after_decimal_point.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+#
+# Statement
+# Given a positive real number, print its first digit to the right of the decimal point.
+
+print("Enter a positive real number")
+n = float(input())
+
+res = int(n * 10) % 10
+
+print("First digit to the right of '.' is: " + str(res) + ".")
```
Comment (kuszcjan) [inline students/sendecki_andrzej/lesson_01_basics/first_digit_after_decimal_point.py]: using `+` for string formatting is OK. However, the preferred way is to use `format`.
Check it out at https://pyformat.info/

File: students/sendecki_andrzej/lesson_01_basics/sum_of_digits.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+print("Enter 3 digit number")
+n = int(input())
+n = abs(n)
+
+d1 = n // 100 % 10
+d2 = n // 10 % 10
+d3 = n % 10
+
+sum = d1 + d2 + d3
```
Comment (kuszcjan) [inline students/sendecki_andrzej/lesson_01_basics/sum_of_digits.py]: Variable name `sum` shadows python built-in function `sum()`
https://docs.python.org/3/library/functions.html
Please avoid that in future.

**Pull request merged**